polkadot/runtime/westend/src/lib.rs

```diff
@@ -1755,6 +1829,7 @@ mod benches {
 		[pallet_multisig, Multisig]
 		[pallet_nomination_pools, NominationPoolsBench::<Runtime>]
 		[pallet_offences, OffencesBench::<Runtime>]
+		[pallet_parameters, Parameters]
```

Comment:
All in all looks good though! will do a final review if the bench bot works as intended. 

@ggwpez this is the first use of this pallet in any of our runtimes? would be good to get your review as well.

Reply:
It is on Rococo, but we did not really use it there yet. I will take a look 😄